.zenodo.json

```diff
@@ -0,0 +1,33 @@
+{
+    "creators": [
+       {
+            "name": "Carsten Ehbrecht",
+            "affiliation": "DKRZ"
+        },
+        {
+            "name": "David Huard",
+            "affiliation": "Ouranos",
+            "orcid": "0000-0003-0311-5498"
+        },
+        {
+            "name": "Trevor James Smith",
+            "affiliation": "Ouranos",
+            "orcid": "0000-0001-5393-8359"
+        },
+        {
+            "name": "Francis Pelletier",
+            "affiliation": "CRIM"
+        },
+        {
+            "name": "David Caron"
```

Comment:
In the JOSS paper for xclim, it was suggested that we add the full name and locations for our affiliations. Would we want to start doing that everywhere (here included)?